packages/design-system-docs/src/pages/startup/migrating-v2.md

```diff
@@ -46,48 +46,60 @@ The `dist` folder structure has been updated for the `@cmsgov/design-system` npm
     └── scss/             Uncompiled CSS
 ```
 
-### Importing SCSS/CSS
+### Fonts and Images
```

Comment:
## Folder structure
### Importing SCSS/CSS
### Importing Javascript
### Fonts and Images
## Focus styles
## Versioning